backend/app/services/collections/create_collection.py

```
@@ -163,29 +134,32 @@ def _mark_job_failed(
-    assistant_crud = None
-    vector_store = None
-    vector_store_crud = None
+    # Keeping the references for potential backout/cleanup on failure
     collection_job = None
+    result = None
+    provider = None
 
     try:
         creation_request = CreationRequest(**request)
+        if (
+            with_assistant == True
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `with_assistant:` for truth checks

Replace with `with_assistant`

(E712)